pystiche_papers/ulyanov_et_al_2016/_utils.py

```diff
@@ -59,6 +84,15 @@ def get_lr(self) -> List[float]:  # type: ignore[override]
 
 
 def lr_scheduler(optimizer: Optimizer, impl_params: bool = True,) -> ExponentialLR:
+    r"""Learning rate scheduler from :cite:`ULVL2016`.
+
+    Args:
+        optimizer: Wrapped optimizer.
+        impl_params: If ``True``, an :class:`~torch.optim.lr_scheduler.ExponentialLR`
+        with a ``gamma of 0.8`` is used instead of a
+        :func:`~pystiche_papers.ulyanov_et_al_2016.DelayedExponentialLR` with a
+        ``gamma of 0.7`` and a ``delay of 5``.
```

Comment:
```suggestion
        with ``gamma==0.8`` is used instead of a
        :func:`~pystiche_papers.ulyanov_et_al_2016.DelayedExponentialLR` with
        ``gamma==0.7`` and ``delay==5``.
```